src/coreclr/vm/jitinterface.h

```diff
@@ -934,6 +952,12 @@ protected :
 
     EEJitManager*           m_jitManager;   // responsible for allocating memory
     CodeHeader*             m_CodeHeader;   // descriptor for JITTED code
+    CodeHeader*             m_CodeHeaderRW;
+    size_t                  m_codeWriteBufferSize;
```

Comment:
This is not code size. It is code header size. Would `m_codeHeaderBufferSize` be a better name?